test/Microsoft.AspNetCore.Mvc.Formatters.Json.Test/JsonInputFormatterTest.cs

```diff
@@ -72,6 +72,51 @@ public class JsonInputFormatterTest
             Assert.Equal(30, userModel.Age);
         }
 
+        [Fact]
+        public async Task BuffersRequestBody_UsingDefaultOptions()
+        {
+            // Arrange
+            var content = "{name: 'Person Name', Age: '30'}";
+            var logger = GetLogger();
+            var formatter =
+                new JsonInputFormatter(logger, _serializerSettings, ArrayPool<char>.Shared, _objectPoolProvider, new MvcOptions());
```

Comment:
We typically put parameters on separate lines rather than breaking the declaration across multiple lines.